app/models/manageiq/providers/automation_manager.rb

```diff
@@ -5,6 +5,8 @@ class ManageIQ::Providers::AutomationManager < ::ExtManagementSystem
   has_many :configured_systems,           :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_profiles,       :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
+  has_many :credentials,                  :class_name => "ManageIQ::Providers::AutomationManager::Authentication",
```

Comment:
so why do we need this relationship definition here again? Isn't it already there in `AuthenticationMixin` [here](https://github.com/jameswnl/manageiq/blob/ab40ebedc07c7ebd06ac4d9f759cafada5bbfdfb/app/models/mixins/authentication_mixin.rb#L5-L5) and `EMS` included that mixin?